setup.py

```diff
@@ -28,7 +28,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/vikinganalytics/mvg",
-    packages=["mvg", "mvg/features"],
+    packages=setuptools.find_packages(),
```

Comment:
```suggestion
    packages=setuptools.find_packages(where="mvg"),
```
Better to limit to the `mvg` folder.